cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -39,16 +42,20 @@ class Global {
 // curl -i -uadmin:password -X POST "http://localhost:8081/artifactory/api/plugins/execute/cleanupCtl?params=command=stop"
 // curl -i -uadmin:password -X POST "http://localhost:8081/artifactory/api/plugins/execute/cleanupCtl?params=command=adjustPaceTimeMS;value=-1000"
 
+
 def pluginGroup = 'cleaners'
+def config = new ConfigSlurper().parse(new File(ctx.artifactoryHome.haAwareEtcDir, PROPERTIES_FILE_PATH).toURL())
 
 executions {
     cleanup(groups: [pluginGroup]) { params ->
         def months = params['months'] ? params['months'][0] as int : 6
         def repos = params['repos'] as String[]
-        def dryRun = params['dryRun'] ? params['dryRun'][0] as boolean : false
-        def disablePropertiesSupport = params['disablePropertiesSupport'] ? params['disablePropertiesSupport'][0] as boolean : false
+        def dryRun = params['dryRun'] ? params['dryRun'][0].toBoolean() : false      
```

Comment:
Why was this change needed?

Reply:
Because for these particular params it was not casted properly. For string, as boolean return true if string is not empty. It was changed because if you put for example dryRun=false in curl it will be true anyway.

Reply:
Thanks a lot. It would make sense to push the toBoolean change to the upstream project as a separate PR since this is definitely an issue